drake/doc/reviewable.rst

```diff
@@ -0,0 +1,57 @@
+****************************************************************
+Tips for Participating In Drake Code Reviews using reviewable.io
+****************************************************************
+
+Drake code reviews use https://reviewable.io. This page documents some
+best practices for communicating effectively in Reviewable.
+
+GitHub Integration
+==================
+
+Avoid using the GitHub UI to comment on code during a review. Reviewable will
+import comments from GitHub, but cannot reliably match them to lines of diff.
+
+When you enter comments in Reviewable, they are saved as drafts. Use the
+"publish" button to send them out in a batch. Reviewable will post the
+comments to the GitHub PR in a single, well-formatted block, generating email
+to everyone else following the PR.
+
+Every time you push to your GitHub branch under review, Reviewable will
+snapshot a new diff. Because it maintains an independent diff series, you can
+rebase freely without corrupting the review history.
+
+Life of a Reviewable Comment
+============================
+
+All threads in Reviewable must be resolved before you can merge your PR. The
+semantics of discussion resolution are a little tricky:
+
+When a reviewer creates a new comment, the reviewer's disposition toward the
+thread may be "satisfied", discussing", or "blocking".
+
+* Use **satisfied** to indicate that no action is required of the author.
```

Comment:
The previous sentence uses the phrase "When a reviewer" whereas these bullets is written as if they were talking to the reviewer. Consider adding "As a reviewer, you should..." before this bullet.